tools/generate_hotkeys.py

```diff
@@ -12,28 +21,51 @@
     }
     for category in HELP_CATEGORIES.keys()
 }
-OUTPUT_FILE = Path(__file__).resolve().parent.parent / "docs" / "hotkeys.md"
-SCRIPT_NAME = PurePath(__file__).name
+OUTPUT_FILE = ROOT_DIRECTORY / "docs" / "hotkeys.md"
+SCRIPT_NAME = pathlib.PurePath(__file__).name
 
 with open(OUTPUT_FILE, "w") as mdFile:
     mdFile.write(
         f"<!--- Generated automatically by tools/{SCRIPT_NAME} -->\n"
         "<!--- Do not modify -->\n\n# Hot Keys\n"
     )
+    # keys_list = [item["keys"] for item in KEY_BINDINGS.values()]
+    # print("keys_list -", keys_list)
     for action in categories.keys():
         mdFile.write(
             f"## {HELP_CATEGORIES[action]}\n"
             "|Command|Key Combination|\n"
             "| :--- | :---: |\n"
         )
+        keys_list = []
+        for help_text, key_combinations_list in categories[action].items():
+            keys_list.append(key_combinations_list)
         for help_text, key_combinations_list in categories[action].items():
             various_key_combinations = " / ".join(
                 [
                     " + ".join([f"<kbd>{key}</kbd>" for key in key_combination.split()])
                     for key_combination in key_combinations_list
                 ]
             )
-            mdFile.write(f"|{help_text}|{various_key_combinations}|\n")
+            for key in keys_list:
+                if key in KEYS_TO_EXCLUDE:
+                    keys_list.remove(key)
+            # Check key combination duplication
+            duplicate_keys = [key for key in keys_list if keys_list.count(key) > 1]
+            if len(duplicate_keys) != 0:
+                print(
+                    f"Duplicate key combination for keys {duplicate_keys} for category ({HELP_CATEGORIES[action]})"
+                )
+                sys.exit(1)
```

Comment:
I suppose duplicate checking need not be done for each `help_text` inside the for loop?